Backends/PDF/class.lisp

```diff
@@ -53,23 +50,15 @@
       "Unknown"))
 
 (defclass clim-pdf-stream (sheet-leaf-mixin
+                           sheet-parent-mixin
                            sheet-mute-input-mixin
                            permanent-medium-sheet-output-mixin
                            sheet-mute-repainting-mixin
-                           ;; ?
-                           mirrored-sheet-mixin
-                           ;; FIXME: Tim Moore suggested (2006-02-06,
-                           ;; mcclim-devel) that this might better be
-                           ;; a superclass of
-                           ;; STANDARD-OUTPUT-RECORDING-STREAM.  This
-                           ;; should be revisited when we grow another
-                           ;; non-interactive backend (maybe a cl-pdf
-                           ;; backend?).  -- CSR.
```

Comment:
please adopt port/graft changes to postscript backend too.

as a side note: we should create an output-stream-backend with base classes meant to be subclassed for both postscript and pdf (possibly also raster) at some point of time.

Reply:
I agree with you that we need an `output-stream-backend`, and for this reason I think that we can mantain the postscript backend without modification now and waiting for future changing. For exampe the new postscript graft could use the graft-*-transformation protocol that you suggest in the next request to define with `defun` because it is local to pdf backend...but if it will be used also by the postscript backend...it will be no more local.

Reply:
I think that the best course of action is to duplicate changes from pdf in postscript backend -- it will be easier to create the output-stream-backend if both backends do not diverge too much.